docs/source/install/k8s_ha.rst

```diff
@@ -301,10 +301,10 @@ By default ``3`` nodes of RabbitMQ are deployed via K8s StatefulSet.
 For more advanced RabbitMQ configuration, please refer to official `rabbitmq-ha <https://github.com/helm/charts/tree/master/stable/rabbitmq-ha>`_
 Helm chart repository, - all settings could be overridden via ``values.yaml``.
 
-etcd
+redis
 ____
```

Comment:
Check the CI failure here:  https://app.circleci.com/pipelines/github/StackStorm/st2docs/278/workflows/623cf438-4d10-406d-9a80-2e8520151f69/jobs/2912

```
Warning, treated as error:
/home/circleci/project/docs/source/install/k8s_ha.rst:305:Title underline too short.
```